server/game/cards/06.6-TBWB/WarriorsBraid.js

```diff
@@ -0,0 +1,34 @@
+const DrawCard = require('../../drawcard.js');
+
+class WarriorsBraid extends DrawCard {
+    setupCardAbilities(ability) {
+        this.whileAttached({
+            effect: [
+                ability.effects.addKeyword('renown'),
+                ability.effects.dynamicStrength(() => this.tokens['bell'])
+            ]
+        });
+        this.reaction({
+            when: {
+                afterChallenge: event => event.challenge.challengeType === 'military' && event.challenge.winner === this.controller &&
+                                         event.challenge.isParticipating(this.parent)
```

Comment:
Card text says that the attached character must be attacking, not just participating.